src/azul/json_freeze.py

```diff
@@ -27,18 +28,34 @@ def freeze(x: AnyJSON) -> AnyJSON:
 
     >>> thaw(freeze(freeze({"1":[2,3]})))
     {'1': [2, 3]}
+
+    `sequence_cls` can be used to avoid producing tuples in the output:
+
+    >>> freeze({"x": [1, 2, 3]}, sequence_cls=frozenset)
+    <frozendict {'x': frozenset({1, 2, 3})}>
```

Comment:
```suggestion
    <frozendict {'x': frozenset({1, 2})}>
```

Reply:
This test might actually fail non-deterministically because the `repr` might also print as `frozenset({2, 1})`. I'm not sure frozenset is a good example to use in the first place but I don't know any other immutable sequence.

Could we subclass `tuple` (just for the doctest) and use that subclass instead of `frozenset`? 

For the next subtest I would use a singleton frozenset.